las/client.py

```diff
@@ -1184,6 +1184,39 @@ def list_workflow_executions(
         }
         return self._make_request(requests.get, url, params=params)
 
+    def update_workflow_execution(
+        self,
+        workflow_id: str,
+        execution_id: str,
+        next_transition_id: str = 'las:transition:commons-failed',
```

Comment:
Do we really want a default argument like this here?

Reply:
I am not sure, I would say it is useful to be able to end an execution without the necessity to read docs, but I see why it might not be ideal as a default argument. Would it be better to just add it to the documentation string?

Reply:
I understand your thought process here, but the fact that 'Updating a workflow execution' defaults to 'Failing a workflow execution' doesn't seem right to me. I suggest leaving it in docstring as you suggested.